lib/src/transformer/declaration_parsing.dart

```diff
@@ -229,15 +223,16 @@ class ParsedDeclarations {
           );
         }
         final isClassPrivate = cd.name.name.startsWith('_');
-        final expectedInitializer = isClassPrivate
-            ? '_\$metaFor${cd.name.name.substring(1)}'
-            : '\$metaFor${cd.name.name}';
+        var expectedInitializers = ['\$metaFor${cd.name.name}', '_\$metaFor${cd.name.name}'];
+        if (isClassPrivate) {
+          expectedInitializers.add('_\$metaFor${cd.name.name.substring(1)}');
+        }
```

Comment:
#nit would be nice to have aTODO or a comment saying which versions need to eventually be removed?

Reply:
I think addressing https://github.com/Workiva/over_react/pull/222#discussion_r246549620 handles that effectively.